extensions/http-curl/sitetosite/HTTPProtocol.cpp

```diff
@@ -286,6 +287,9 @@ void HttpSiteToSiteClient::closeTransaction(const std::string &transactionID) {
   logger_->log_debug("Received %d response code from delete", client->getResponseCode());
 
   if (client->getResponseCode() == 400) {
+    std::string error(client->getResponseBody().data(), client->getResponseBody().size());
+
+    logger_->log_warn("400 received: %s", error);
```

Comment:
You could very easily hit the inner logbuffer's 1024 byte limitation and truncate the error here. I would suggest using 
`logging::LOG_WARN(logger_) <<`

Reply:
Good shout, thanks!